boost-install.jam

```diff
@@ -479,19 +493,6 @@ rule generate-cmake-config- ( target : sources * : properties * )
             ""
```

Comment:
You need to move the `include(CMakeFindDependencyMacro)` above the variant includes.